storage/innobase/xtrabackup/doc/source/release-notes/2.3/2.3.9.rst

```diff
@@ -0,0 +1,44 @@
+==========================
+|Percona XtraBackup| 2.3.9
+==========================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.3.9 on
+July 24th 2017. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.3.9/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+This release is the current GA (Generally Available) stable release in the 2.3
+series.
+
+New Features
+============
+
+ New :option:`xtrabackup --check-privileges` option has been implemented that
+ can be used to check if |Percona XtraBackup| has all
+ :ref:`required privileges <privileges>` to perform the backup.
+
+Bugs fixed
+==========
+
+ |Percona XtraBackup| would crash when being prepared if the index compaction
+ was enabled. Bug fixed :bug:`1192834`.
+
+ Fixed build failure on *Debian Stretch* by adding support for building with
+ *OpenSSL* 1.1. Bug fixed :bug:`1678947`.
+
+ Streaming backups from a database containing many tables via |xbstream| would
+ fail because the receiving |xbstream| process running out of file descriptors.
+ Bug fixed :bug:`1690823`.
+
+ |Percona XtraBackup| incorrectly determined GTID mode on a slave, and
+ thus generated invalid :file:`xtrabackup_slave_info` file. Bug fixed
+ :bug:`1705193`.
+
+ |Percona XtraBackup| would crash while backing up MariaDB 10.2.x with
+ ``--ftwrl-*`` options. Bug fixed :bug:`1704636`.
+
+ :option:`xtrabackup --extra-lsndir` option now includes the
```

Comment:
Along with "xtrabackup_checkpints" xtrabackup now copies "xtrabackup_info" file into directory specified by --extra-lsndir. Because option cannot include the file.